File: python/week07/Week07 Python/Circle.py
from Point import Point
import math


class Circle(Point):

    def __init__(self, x, y, r):
        super().__init__(x, y)
        self.r = r

    def __str__(self):
        return f"R={self.r} Center=" + super().__str__()

    def area(self):
        return math.pi * self.r*self.r

File: Instructor Java/Week07  Java/Week07 Java/Circle.java
/**
 * Circle
 */
public class Circle extends Point {

  private int r = 0;

  public Circle(int x, int y, int r) {
    super(x, y);
    this.r = r;
  }

  /**
   * @return the r
   */
  public int getR() {
    return r;
  }

  /**
   * @param r the r to set
   */
  public void setR(int r) {
    this.r = r;
  }

  @Override
  public double area() {
    return Math.PI * r * r;
  }

  @Override
  public String toString() {
    return String.format("R=%d Center=(%d,%d)", r, this.getX(), this.getY());
  }

}
